tf_encrypted/keras/layers/pooling.py

```diff
@@ -167,3 +167,104 @@ def __init__(self,
         self.prot.avgpool2d,
         pool_size=pool_size, strides=strides,
         padding=padding, data_format=data_format, **kwargs)
+
+class GlobalPooling2D(Layer):
```

Comment:
should this be a proper abstract base class from the `abc` module? in that case this would need to inherit from `abc.ABC` and `call` would need to be decorated with `@abstractmethod`

Reply:
@jvmancuso done